bin/tinyreq

```diff
@@ -26,17 +27,20 @@ new clp({
   , examples: [
         "tinyreq -u http://ionicabizau.net"
     ]
-}, [urlOpt, methOpt, dataOpt, fieldsOpt]);
+}, [urlOpt, methOpt, dataOpt, fieldsOpt, quietOpt]);
 
 // Merge the options
 let options = ul.merge({
     method: methOpt.value || undefined
   , url: urlOpt.value || undefined
   , data: dataOpt.value || undefined
+  , quiet: quietOpt.value = quietOpt.is_provided
 }, {
     method: "GET"
 });
-
+if (options.quiet) {
+  logger.config.level = -1;
```

Comment:
If you say `-1`, all the logs will be logged, because their levels is greater than `-1`. So, if you set a value like `5`, no log will appear, but to be safe, you can set `Infinity`. 😁